mathics/builtin/attributes.py

```diff
@@ -476,9 +476,14 @@ class OneIdentity(Predefined):
 
     'OneIdentity' affects pattern matching:
     >> SetAttributes[f, OneIdentity]
-    >> a /. f[args___] -> {args}
+    >> a /. f[x_:0, u_] -> {u}
```

Comment:
Reading the documentation, I realize that this test was wrong...

Reply:
Good catch! This bug goes back to the very first Mathics version.